modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1519,9 +1519,56 @@ public void testUpdateCustomObjectDefinition() throws Exception {
 		_testUpdateCustomObjectDefinitionThrowsObjectFieldRelationshipTypeException(
 			objectDefinition);
 
+		_assertFailure(
+			ObjectDefinitionExternalReferenceCodeException.
+				MustNotStartWithPrefix.class,
+			"The prefix L_ is reserved for Liferay", objectDefinition,
+			currentObjectDefinitionId ->
+				_objectDefinitionLocalService.updateCustomObjectDefinition(
+					"L_TEST_ERC", currentObjectDefinitionId, 0, 0, 0, false,
+					true, true, false, false, true,
+					LocalizedMapUtil.getLocalizedMap("Baker"), "Baker", null,
+					null, false, LocalizedMapUtil.getLocalizedMap("Bakers"),
+					ObjectDefinitionConstants.SCOPE_COMPANY));
+
```

Comment:
Hey @4lejandrito isn't this the same check of https://github.com/liferay-objects/liferay-portal/blob/c312eb407e7925ba95a64edea04f1d17906e1914/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java#L1398-L1405 ?

Am I missing something?